CONTRIBUTING.md

```diff
@@ -60,18 +60,17 @@ This will host an interactive editor at
 [http://localhost:7936/](http://localhost:7936/) to play around with and test
 changes.
 
-#### Jasmine tests
+#### Jest tests
 
-The JavaScript parser and some of the tree
-builder is tested with Jasmine. These tests can be run either using node with
-`make test`, or in the browser by visiting
-[http://localhost:7936/test/test.html](http://localhost:7936/test/test.html).
```

Comment:
@xymostech I ended up linking to the jest troubleshooting docs which should be a good starting spot.  Unfortunately they suggest `node-inspector` which I've found cumbersome.  I've used `vscode`'s built-in debugger for debugging node code in the past but I've had to fight with it to debug tests and not everyone uses `vscode` and I don't want to ask people to install it just to debug something.  Do you know of any better alternatives to `node-inspector`?  (maybe it's gotten better since the last time I used it)